governance/votes/Executive vote - November 2, 2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022
+summary: Maker Teleport Technical Deployment on Starknet, Golden Parachute MKR and DAI budget transfers for EVENTS-001, SH-001, and RWF-001, yank MKR and DAI budget streams for EVENTS-001, SH-001, and RWF-001, whitelist Oasis.app on rETH/USD oracle, and BlockTower Legal Work and Risk Work SPF Funding. 
+date: 2022-11-02T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Maker Teleport Global [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased by **1 million DAI** to **3 million DAI**.
```

Comment:
This is just a summary it's ok to be less detailed here

```suggestion
- The Maker Teleport Global [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased, as detailed below.
```

Reply:
I've committed your change but just checking that the global debt ceiling for Teleport is now 3 million and not 2,100,000? Just thought that might be worth clarifying